src/overlays/actors/ovl_En_Aob_01/z_en_aob_01.c

```diff
@@ -110,12 +110,12 @@ void func_809C1158(EnAob01* this, PlayState* play) {
     s32 temp_s0 = ENAOB01_GET_7E00_2(&this->actor);
     s16 i = 0;
 
-    if (temp_s0 != 0x3F) {
+    if (temp_s0 != PATH_INDEX_MAX) {
         do {
-            this->unk_1D8[i] = SubS_GetPathByIndex(play, temp_s0, 0x3F);
-            temp_s0 = this->unk_1D8[i]->unk1;
+            this->unk_1D8[i] = SubS_GetPathByIndex(play, temp_s0, PATH_INDEX_MAX);
+            temp_s0 = this->unk_1D8[i]->additionalPathIndex;
```

Comment:
```suggestion
            pathIndex = this->paths[i]->additionalPathIndex;
```

Reply:
Aob01 is weird. It isn't her path, but related to the dogs. I'd rather wait for more docs around the whole thing which darkeye has been working on (https://discord.com/channels/688807550715560050/873211000122384415/1080652253498716190)